doc/setup.md

```diff
@@ -21,21 +21,28 @@ $ elan default leanprover/lean4:nightly
 $ elan override set leanprover/lean4:stable
 ```
 
-### `leanpkg`
+NOTE: there are no `lean4:stable` releases yet, so for now
+just use `leanprover/lean4:nightly`
 
-Lean 4 comes with a basic package manager, `leanpkg`.
-Use `leanpkg init Foo` to initialize a Lean package `Foo` in the current directory, and `leanpkg build` to typecheck and build it as well as all its dependencies; call `leanpkg help` to learn about further commands.
+### `lake`
+
+Lean 4 comes with a package manager named `lake`.
+Use `lake init Foo` to initialize a Lean package `Foo` in the current directory, and `lake build` to typecheck and build it as well as all its dependencies; call `lake help` to learn about further commands.
 The general directory structure of a package `Foo` is
 ```sh
-leanpkg.toml  # package configuration
-Foo.lean  # main file, import via `import Foo`
+lakefile.lean  # package configuration
+lean-toolchain # specifies the lean version to use
+Foo.lean       # main file, import via `import Foo`
 Foo/
-  A.lean  # further files, import via e.g. `import Foo.A`
-  A/...   # further nesting
-build/  # `leanpkg` output directory
+  A.lean       # further files, import via e.g. `import Foo.A`
+  A/...        # further nesting
+build/         # `lake` build output directory
+```
+
+After running `lake build` you will see a binary named `.\build\bin\Foo` and when you run it you should see the output:
```

Comment:
```suggestion
After running `lake build` you will see a binary named `./build/bin/Foo` and when you run it you should see the output:
```
I believe that will work on both Windows and Linux.

Reply:
woops, fixed.